src/Akka.Cluster.Hosting/AkkaClusterHostingExtensions.cs

```diff
@@ -367,5 +372,47 @@ public static class AkkaClusterHostingExtensions
                 CreateAndRegisterSingletonProxy<TKey>(singletonName, singletonManagerPath, singletonProxySettings, system, registry);
             });
         }
+
+        /// <summary>
+        /// Configures a <see cref="ClusterClientReceptionist"/> for the <see cref="ActorSystem"/>
+        /// </summary>
+        /// <param name="builder">The builder instance being configured.</param>
+        /// <param name="configure">An action that can be used to configure the receptionist settings</param>
+        /// <returns>The same <see cref="AkkaConfigurationBuilder"/> instance originally passed in.</returns>
+        public static AkkaConfigurationBuilder WithClusterClientReceptionist(
+            this AkkaConfigurationBuilder builder,
+            Action<ClusterClientReceptionistOptions> configure = null)
```

Comment:
Will this successfully start a `ClusterClientReceptionist` by default?

Reply:
Yes, it would, everything would be at the default configuration if nothing is provided.